### PR TITLE
Remove Global Scroll on Document Edit Page

### LINKED ISSUE #201: Remove Global Scroll on Document Edit Page
**What would you like to be added**:
On the document edit page, the global scroll should not be visible. Currently, due to unknown margins outside the document, the global scroll is visible.

**Why is this needed**:
To provide a cleaner look and better user experience on the document edit page.

**Additional Information**:

### PR DESCRIPTION
**Pull request by bishoe01**
**What this PR does / why we need it**:

This pull request to prevent users from experiencing global scrolling with bottom whitespace, which was intermittently caused by differences in the height of markdown syntax, such as latex, and the posts written in it.

**Which issue(s) this PR fixes**:

Fixes #201

**Special notes for your reviewer**:
encountered an issue when editorStore.mode was set to edit or both. We initially attempted to resolve the problem by removing the height: "100%" value from the Box component in each mode. While this approach worked for the edit mode, it caused an issue in the "both mode" where individual block scrolling was lost. To address this, we added ```overflow: hidden``` to the body of the top-level html element, which resolved the issue and ensured that it worked correctly in other scenarios as well.

If you have any suggestions for a better solution or notice any issues with this approach, we would greatly appreciate your feedback.

**Does this PR introduce a user-facing change?**:

With overflow:hidden, we wanted to prevent users from experiencing global scrolling with bottom whitespace, which was intermittently caused by differences in the height of markdown syntax, such as latex, and the posts written in it.

**Checklist**:
- [x] Added relevant tests or not required
- [x] Didn't break anything

## Summary by CodeRabbit

- **Style**
	- Updated the body element's overflow behavior to enhance layout control by preventing content overflow visibility.

### CODE REVIEW

**Comment by devleejb** [comment]
@bishoe01 

> [CLA assistant check](https://cla-assistant.io/yorkie-team/codepair?pullRequest=243) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/yorkie-team/codepair?pullRequest=243) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/yorkie-team/codepair?pullRequest=243) it.

Please check Contributor License Agreement.

File: frontend/src/index.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 body {
 	margin: 0;
 	min-height: 100vh;
+    overflow: hidden;
```
Comment (devleejb) [inline frontend/src/index.css:4]: It seems that it needs running code formatting.

**Review by devleejb — CHANGES_REQUESTED** [review]

File: frontend/src/index.css (hunk 1 above)
Comment (bishoe01) [reply frontend/src/index.css]: I made a mistake. I finished formatting the code.

**Comment by bishoe01** [comment]
I agree. In the future, it would be nice to consider a more fundamental approach to this problem - perhaps a solution like dynamically resizing fonts in situations with large height differences would make the layout more flexible. Thank you for your review and feedback.

**Pull request merged**